tools/cli/commands/docker.js

```diff
@@ -724,8 +738,8 @@ export function dockerDefine( yargs ) {
 					handler: argv => execDockerCmdHandler( argv ),
 				} )
 				.command( {
-					command: 'phpunit-wpcomsh',
-					alias: 'phpunit:wpcomsh',
+					command: 'phpunit-jp-wpcomsh',
+					alias: 'phpunit:jp:wpcomsh',
```

Comment:
Given this alias has never existed, perhaps we don't need it?

Reply:
🤷